internal/wrappers/feature-flags.go

```diff
@@ -85,7 +132,7 @@ func loadFeatureFlagsMap(allFlags FeatureFlagsResponseModel) {
 	}
 }
 
-func loadFeatureFlagsDefaultValues() {
+func LoadFeatureFlagsDefaultValues() {
```

Comment:
why do you change the func to public? (Load instead of load)

Reply:
to test this func